config.py

```diff
@@ -13,6 +11,7 @@ def env_path(environment):
 
 
 class Config:
+    PORT = os.getenv('PORT')  # This is not used in run.py which isn't used in cloudfoundry
```

Comment:
Probably should read: 

`# This is *only* used in run.py which isn't used in cloudfoundry`